hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/container/InfoSubcommand.java

```diff
@@ -52,6 +53,10 @@
   @Parameters(description = "Decimal id of the container.")
   private long containerID;
 
+  @CommandLine.Option(names = {"--verbose"},
```

Comment:
Can we leverage the existing "--verbose" option defined in parent class GenericCli here?

Reply:
@xiaoyuyao 
```java
boolean verbose = false;
if (parent.getParent() instanceof GenericCli) {
        if (((GenericCli) parent.getParent()).isVerbose()) {
            verbose = true;
        }
}
```

Do you mean that i should add this code to L66

Reply:
I agree uses GenericParentCommand seems better. Also, I would suggest we change printError to printStackTrace only if there is no meaningful error message? This way, we will have a clean verbose output as long as the error message is meaningful.

Reply:
@adoroszlai Thanks for your tips, I commit a new PR, PTAL.

Reply:
> Also, I would suggest we change printError to printStackTrace only if there is no meaningful error message? This way, we will have a clean verbose output as long as the error message is meaningful.

Good idea @xiaoyuyao.  I think we can split this to a separate improvement issue, so I created HDDS-4160.

Reply:
Ping @xiaoyuyao.

Please let me know if you are happy with this approach.